README.md

```diff
@@ -2,3 +2,4 @@
 
 # Add your favorite dessert here!
 
+# Unknown
```

Comment:
Thanks so much for the PR. If you could please add a dessert and no need to use a # before it. Thanks!

Reply:
Thanks for the guidance. I'll make the necessary changes.